opportunities/index.html

```diff
@@ -165,25 +155,32 @@ <h2 class="styled">Training &amp; Development <span></span></h2>
                         <div class="grid width-33 pad-md">
-                            <div class="grid-item wow fadeInUp" data-wow-delay="0.6s">
+                            <div class="grid-item wow fadeInUp" data-wow-delay="0.3s">
                                 <div class="box-image"><p class="max-width-100"><img src="../media/k0hg3sp5/lfx98b0.png?mode=max&amp;height=50"></p></div>
                                 <h4><a rel="noopener" href="https://training.linuxfoundation.org/training/kubernetes-and-cloud-native-essentials-lfs250/" target="_blank">KCNA Essentials Course</a></h4>
                                 <p>This course provides an overview of cloud native technologies and how container orchestration systems like Kubernetes can help to implement and maintain them.</p>
                                 <p><em>*</em><em>Ask about sponsored access for Māori cohorts</em></p>
-                            </div><div class="grid-item wow fadeInUp" data-wow-delay="0.9s">
+                            </div><div class="grid-item wow fadeInUp" data-wow-delay="0.4s">
                                 <div class="box-image"><p class="max-width-100"><img src="../media/km1dhcr1/kcna-logo-300x300a047.png?mode=max&amp;height=130"></p></div>
                                 <h4><a rel="noopener" href="https://training.linuxfoundation.org/certification/kubernetes-cloud-native-associate/" target="_blank">Kubernetes Cloud Native Associate (Exam)</a></h4>
                                 <p>The KCNA exam demonstrates a user’s foundational knowledge and skills in Kubernetes and the wider cloud native ecosystem.</p>
                                 <p><em>*</em><em>Ask about sponsored access for Māori cohorts</em></p>
-                            </div><div class="grid-item wow fadeInUp" data-wow-delay="1.2s">
+                            </div><div class="grid-item wow fadeInUp" data-wow-delay="0.5s">
                                 <div class="box-image"><p class="max-width-100"><img src="../media/tjkkzd4q/edx98b0.png?mode=max&amp;height=50"></p></div>
                                 <h4><a rel="noopener" href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank">EDX Essentials Course</a></h4>
                                 <p>Want to learn Kubernetes for free? Get an in-depth primer on this powerful system for managing containerised applications.</p>
                             </div>            </div>
                     </div>
                 </div>
             </section>
-
+            <section>
+                <div class="container">
+                    <h2 class="styled">Good First Issues <span></span></h2>
+                    <p>A good first issue is a label that can be set on Issues, usually these are easy to navigate, have heaps of instruction and direction on how to implement the needed changes</p>
+                    <p>Get involved with open source today by contributing to this website</p>
```

Comment:
Hyperlink to contributing instructions

Reply:
Will do this when we have a contribution guide